content/docs/2.6/scalers/datadog.md

```diff
@@ -41,6 +41,8 @@ You should use `TriggerAuthentication` CRD to configure the authentication:
 
 ### Example
 
+The example below uses the default KEDA polling interval (30 seconds). Take into account that [API Datadog endpoints are rate limited](https://docs.datadoghq.com/api/latest/rate-limits/) and reducing the polling interval can accelerate reaching it. If your account has reached its rate limit, a relevant error will be logged in KEDA.
```

Comment:
Might be good to make this a more prominent piece by moving it to the top of the article?

For example:
> :lightbulb: Consider the [API Datadog endpoints are rate limited](https://docs.datadoghq.com/api/latest/rate-limits/) when defining polling interval

Reply:
done